phases/stack.go

```diff
@@ -63,8 +63,9 @@ func Stack(projectType string) (string, error) {
 	}
 
 	systemReportURL := fmt.Sprintf("https://github.com/bitrise-io/bitrise.io/blob/master/system_reports/%s.log", stack)
+	log.Printf("A(n) %s project has been detected based on the bitrise.yml", projectType)
```

Comment:
Print `log.Printf("Project type: ", projectType)` so it is valid both for scanner and preexisting bitrise.yml.

Reply:
yes

Reply:
it can be green (the projecttype) if you want to be consistent